pkgs/games/xivlauncher/default.nix

```diff
@@ -40,6 +42,7 @@ in
     postFixup = lib.optionalString useSteamRun ''
       substituteInPlace $out/bin/XIVLauncher.Core \
         --replace 'exec' 'exec ${steam-run}/bin/steam-run'
+      wrapProgram $out/bin/XIVLauncher.Core --prefix GST_PLUGIN_SYSTEM_PATH_1_0 ":" "$GST_PLUGIN_SYSTEM_PATH_1_0"
```

Comment:
Putting this here makes it conditional on `useSteamRun`. Is that correct or should it be unconditionally done? (I don't care either way and don't use this package -- I just wanted to double check before merge)

Reply:
ugh, good point – it should probably be unconditional, I'll fix that.